app/database_etl/location_utils/country_alternative_names.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
why do we need 2 alt names? Do the deaths and cases dfs use diff country names???

Reply:
No, but we have different names in Airtable